app/views/employments/EndEmploymentView.scala.html

```diff
@@ -15,41 +15,89 @@
  *@
 
 @import includes.forms._
-@import includes.{cancelLink, ptaHeader}
+@import includes.{cancelLink}
 @import java.time.LocalDate
-@import uk.gov.hmrc.play.views.html.helpers._
+
 @import uk.gov.hmrc.tai.viewModels.employments.EmploymentViewModel
 
 @import scala.concurrent.ExecutionContext
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukDateInput
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcPageHeadingLegend
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcSectionCaption
+
+@this(mainTemplate: MainTempleteNew, form: FormWithCSRF,  govukDateInput : GovukDateInput, govukButton: GovukButton, govukErrorSummary: GovukErrorSummary)
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
+@(dateForm: Form[LocalDate], viewmodel: EmploymentViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser,  ec: ExecutionContext)
 
-@(dateForm: Form[LocalDate], viewmodel: EmploymentViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,  ec: ExecutionContext)
 
-@header = @{
-    ptaHeader(
-        formForErrorSummary = Some(dateForm),
-        displayBackLink = true,
-        mainHeadingText = Messages("tai.endEmployment.endDateForm.title", viewmodel.employerName),
-        preHeadingText = Messages("tai.endEmployment.preHeadingText")
-    )
+@hint = {
+    <div>
```

Comment:
is this div needed? i think the Hint puts everything inside one (i might be wrong)